services/web/src/utils/README.md

```diff
@@ -91,3 +150,11 @@ import { urlForUpload } from 'utils/uploads';
 
 urlForUpload(uploadObject);
 ```
+
+## URLS
+
+```javascript
+import { getDocumentLocationQuery } from 'utils/urls';
```

Comment:
ok hard to not sound like an asshole here but..... it's 2021... this is now as simple as:

```js
new URLSearchParams(document.location.search).get('gclid')
```

Reply:
2021 called they want their URLSEARCHPARAMS BACK

Reply:
deleted it all